### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.servicebus.common.errors.ServiceBusAuthorizationError
  fullName: azure.servicebus.common.errors.ServiceBusAuthorizationError
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - type: builtins.object
        type: builtins.BaseException
      type: builtins.Exception
    type: azure.servicebus.common.errors.ServiceBusError
  langs:
  - python
  module: azure.servicebus.common.errors
  name: ServiceBusAuthorizationError
  source:
    id: ServiceBusAuthorizationError
    path: azure-servicebus\azure\servicebus\common\errors.py
    remote:
      branch: servicebus
      path: azure-servicebus\azure\servicebus\common\errors.py
      repo: https://github.com/annatisch/azure-sdk-for-python
    startLine: 106
  summary: 'An error occured when authorizing the connection.

    '
  syntax:
    content: ServiceBusAuthorizationError(message, inner_exception=None)
  type: class
  uid: azure.servicebus.common.errors.ServiceBusAuthorizationError
references: []
